### ISSUE TITLE
Drag and Drop is not supported in JavaScript

### ISSUE THREAD

**Issue by Manukaki** [shannah/cn1-native-data-transfer#2]
Hi,
I have try on this DnD and found it is working on Simulator, but not in Javascript port.

After sent to Javascript build, the DropTarget.isSupported() just return false, I put an else {} to print out the message.

I have prevent the browser to load the drag file by into the index.html

> `<script>
> window.addEventListener("dragover",function(e){
>   e = e || event;
>   e.preventDefault();
> },false);
> window.addEventListener("drop",function(e){
>   e = e || event;
>   e.preventDefault();
> },false);        
> </script>`

and successfully force the CN form triggered the DropTarget.isSupported(), but just return false.

Could you please advice is Javascript port really support this cn1-native-data-transfer?

Thank you.

**Comment by shannah** [comment]
Yes JavaScript is supported.  Sounds like the library didn't get included properly.  Are you doing this in a fresh project or is the project older?

**Comment by shannah** [comment]
Also which ide are you using?

**Comment by Manukaki** [comment]
I am using Netbeans for both.

Yes, I add in into an old/active project. After I download and unzip the
build Javascript apps, I did not see any extra cnlib files included inside
the zip file. I also guess it was not get included.

I actually download your DND code and modified the function
isMimeAccepted(String mime), add an else

```
} else if (mimetype.indexOf("text/") == 0) {
            return true;
}
```

to let my apps to accept text file upload.

After compile the newly create cnlib, I copy it and put under my projects's
lib/ folder.

I tried to run in the simulator, and it is working as expected. I can read
the text file drop on the form directly into my apps.

But if I sent to Javascript build, it is not working. The function
DropTarget.isSupported() is just return false.

Kindly advice.

**Comment by Manukaki** [comment]
Any idea to fix..? Kindly advice.

**Comment by shannah** [comment]
I'm unable to reproduce this.  Try building the demo app.  Does that work for you?
